### DISCUSSION TITLE
Why Flatpak shows very old version  number?

### DISCUSSION THREAD

**Discussion by lamyergeier** [ubuntu-flutter-community/musicpod#792] [Q&A] [answered]
Shows version 0.3.7!

![image](https://github.com/ubuntu-flutter-community/musicpod/assets/42092626/8749093d-bd04-43f1-9bae-6e5e7a0fdf55)

**Comment by Feichtmeier** [comment]
@TheShadowOfHassen could you take a look at this please if you have time

**Comment by TheShadowOfHassen** [comment]
Very simple answer. I've been forgetting to bump the version number on the flatpak file. The way it's currently set up, it's a lot of work. I'm going to work on a PR to make it a little easier, and hopefully after the next release it'll happen a bit more frequently.

**Reply by Feichtmeier** [reply]
Thank you!

**Comment by TheShadowOfHassen** [comment, marked as answer]
I made a fix that'll make it easier to update. The version number bump might take a while but one should be coming.
